src/features/organizations/hooks/useOrganizationsTree.tsx

```diff
@@ -0,0 +1,28 @@
+import getUserOrganizationsTree from 'features/organizations/rpc/getUserOrgTree';
+import { loadListIfNecessary } from 'core/caching/cacheUtils';
+import { RootState } from 'core/store';
+import { useSelector } from 'react-redux';
+import { treeDataLoad, treeDataLoaded } from '../store';
+import { useApiClient, useEnv } from 'core/hooks';
+
+const useOrganizationsTree = () => {
```

Comment:
I see other hooks have return types, but it is missing here. I think it would be nice to have return type here to unify the pattern